model-inference/decisionTree/experiments/train_model.py

```diff
@@ -111,13 +93,14 @@ def train(config, df_train):
 
     # Save the model using joblib
     joblib_time_start = time.time()
-    joblib.dump(model, get_relative_path("models", f"{DATASET}_{MODEL}_{config['num_trees']}_{config['depth']}.pkl"))
+    joblib.dump(model, relative2abspath("models", f"{DATASET}_{MODEL}_{config['num_trees']}_{config['depth']}.pkl"))
     joblib_time_end = time.time()
-    print(f"Time taken to save model using joblib: {calulate_time(joblib_time_start, joblib_time_end)}")
+    print(f"Time taken to save model using joblib: {calculate_time(joblib_time_start, joblib_time_end)}")
 
 
 if __name__ ==  "__main__":
     parse_arguments()
-    config = json.load(open(get_relative_path("config.json")))
-    df_train = load_data(config)
+    config = json.load(open(relative2abspath("config.json")))
+    df_train = df_train = fetch_data(DATASET,config,"train")
```

Comment:
Is this duplicate assignment intentional?

Reply:
Good catch. It doesn't hurt but is unnecessary. I will correct it soon.